services/QuillLMS/app/controllers/pages_controller.rb

```diff
@@ -467,7 +467,7 @@ def backpack
     @style_file = 'staff'
   end
 
-  def comprehension
+  def evidence
     allow_iframe
     @style_file = ApplicationController::COMPREHENSION
```

Comment:
Does this need to be updated?  It looks like the answer is probably "no" (same thing on line 524 below), but I wanted to double check.

Reply:
Same as above!